### ISSUE TITLE
Please add "Launch at Login"

### ISSUE THREAD

**Issue by FrediWest** [kyleneideck/BackgroundMusic#466]

**Comment by kyleneideck** [comment]
Good idea. It should be simple enough to add that to the Preferences menu.

Until someone gets around to that, you can set it up manually in `System Preferences > Users & Groups > Login Items`. (Details: <https://support.apple.com/en-US/guide/mac-help/mh15189/mac>)

**Comment by FrediWest** [comment]
> Good idea. It should be simple enough to add that to the Preferences menu.
> 
> Until someone gets around to that, you can set it up manually in `System Preferences > Users & Groups > Login Items`. (Details: https://support.apple.com/en-US/guide/mac-help/mh15189/mac)

thank you, for the temporary fix suggestion, that helped.

**Comment by crisbal** [comment]
Hi, if the system is managed by an organization, `Users & Groups` can be grayed out and `Login Items` can't be accessed.

See here:

<img width="186" alt="image" src="https://user-images.githubusercontent.com/2728038/201860179-1e2a888a-9626-4728-a6af-002a5486d5e3.png">

Other software can start at login just fine, but they do the setup automatically (usually in preferences)
